### PR TITLE
I implemented chat bubble colours on SematicColour file

### PR DESCRIPTION
**Pull request by leeashlee**
## Type of change

- [ ] Feature
- [ ] Bugfix
-  Technical
- [ ] Other :

## Content

I made Material You as true and put chat bubble colours inside SematicColours! :)

## Tested devices

- [ ] Physical
-  Emulator
- OS version(s):

## Checklist

-  Changes have been tested on an Android device or Android emulator with API 23
-  UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
-  Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-x-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
-  You've made a self review of your PR

### CODE REVIEW

**Comment by bmarty** [comment]
Thanks for the PR.

> I made Material You as true and put chat bubble colours inside SematicColours! :)

The content does not match the description. It looks like this PR adds support for sticker messages (which we will definitely review later). Can you check please?

**Comment by seankhl** [comment]
I think defaulting to "element-branded" colors is fine, but I would greatly appreciate and support an option for Material You colors. At the end of the day I want element to feel native on my phone and I think for a good number of Android users it's a make or break feature. Maybe that's a way to achieve the best of both worlds?

**Comment by bmarty** [comment]
@seankhl can I ask you to post again your comment in https://github.com/vector-im/element-x-android/issues/1265 to make it visible to the product owner? Thanks.

File: app/build.gradle.kts [hunk 1]
```diff
@@ -38,7 +38,7 @@ android {
     namespace = "io.element.android.x"
 
     defaultConfig {
-        applicationId = "io.element.android.x"
+        applicationId = "ashlee.animent.android.x"
```
Comment (bmarty) [inline app/build.gradle.kts:41]: It looks like you are creating an alternative application. That's fine, but the PR does not need to exist.

**Pull request closed without merging**